docs/development/design/shared-cidr.md

```diff
@@ -0,0 +1,32 @@
+# Using a single CG-NAT IPv4 Address space by default
+
+> [Issue #1011](https://github.com/nexodus-io/nexodus/issues/1011)
+
+## Summary
+
+Currently, each org gets a new IPAM address space for IP address allocation. This means the same IP address could be allocated to devices in different organizations. This would prevent a device in from being shared with other organizations.
+
+Instead, organizations should default to using the shared CG-NAT address space which is 100.64.0.0/10, i.e. IP addresses from 100.64.0.0 to 100.127.255.255. This is about 4 Million addresses.
+
+## Proposal
+
+Why we should be using a single CG-NAT IPv4 Address space in Nexodus:
+
+1. Giving devices a stable IP identity is paramount.  This identity should be preserved even if the device is moved from one organization to another organization.
```

Comment:
How would this work when some organizations use the shared space and some use a custom CIDR?

Reply:
This is the same as asking how would different TMobile customers (aka organizations) use the shared TMobile CG-NAT addresses assigned to their mobile phones.

Reply:
but if a device is moved from an org using the shared CG-NAT space to an org using a different, private CIDR, then it can't take its IP with it. I believe that's what I was trying to get at.

Reply:
yeah.. correct.. movement would only be allowed if your IPAM namespace is not changing.